### PR TITLE
Add `conv_transpose` into reference operations

### PR DESCRIPTION
**Pull request by taehoonlee**
This PR adds `conv_transpose` and `dilated_conv_transpose`  into numpy reference operations so that we can reduce CI build time.

### CODE REVIEW

File: tests/keras/backend/backend_test.py [hunk 1]
```diff
@@ -1064,6 +1049,20 @@ def test_conv(self, op, input_shape, kernel_shape, padding, data_format):
...
+            ('conv2d_transpose', (2, 5, 6, 3), (3, 3, 2, 3), (2, 5, 6, 2),
+             'same', 'channels_last'),
+            ('conv2d_transpose', (2, 3, 8, 9), (3, 3, 2, 3), (2, 2, 8, 9),
+             'same', 'channels_first'),
+        ])
+    def test_conv_transpose(self, op, input_shape, kernel_shape, output_shape,
+                            padding, data_format):
+        check_two_tensor_operation(
+            op, input_shape, kernel_shape, WITH_NP,
+            output_shape=output_shape,padding=padding, data_format=data_format,
```
Comment (gabrieldemarmiesse) [inline tests/keras/backend/backend_test.py]: There is a space missing before `padding`. I'm surprised that Travis didn't catch that when running the pep8 tests.

Comment (gabrieldemarmiesse) [reply tests/keras/backend/backend_test.py]: It seems that all pep8 tests failures are ignored by travis. I'll make a PR to fix it.

Comment (taehoonlee) [reply tests/keras/backend/backend_test.py]: Updated. Thank you for the comment.

**Comment by gabrieldemarmiesse** [comment]
Can you merge master into your branch to make sur there is no other issue with PEP8?

**Comment by taehoonlee** [comment]
Thank you for the comment, @gabrieldemarmiesse. Merged with the master.

**Comment by gabrieldemarmiesse** [comment]
Thanks for the PR!

**Pull request merged**